tools/group_humann2_uniref_abundances_to_GO/group_humann2_uniref_abundances_to_go.xml

```diff
@@ -0,0 +1,125 @@
+<tool id="group_humann2_uniref_abundances_to_go" name="Group abundances" version="1.2.1">
+    <description> of UniRef50 gene families obtained (HUMAnN2 output) to Gene Ontology (GO) slim terms</description>
+
+    <requirements>
+        <requirement type="package" version="1.9">numpy</requirement>
+        <requirement type="package" version="0.6.4">goatools</requirement>
+        <requirement type="package" version="1.2.1">group_humann2_uniref_abundances_to_go</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" />
+        <regex match="IOError"
+               source="both"
+               level="fatal"
+               description="Python file error" />
+        <regex match="pkg_resources"
+               source="both"
+               level="fatal"
+               description="PKG Error" />
+    </stdio>
+
+    <version_command></version_command>
```

Comment:
can you fill this please out if possible?

Reply:
Not possible with the current script. I took it out for now.